docs/api/cozy-client.md

```diff
@@ -1260,47 +1257,50 @@ attributes used to describe the document.
 The qualifications model is stored in the assets, associating
 labels to attributes, namely: purpose, sourceCategory, sourceSubCategory
 and subjects.
-Only subjects can be customized: see the checkQualification for more
-details about the qualification rules.
+A qualification can be customized accordingly to rules detailed in
+the checkValueAttributes method.
 
 **Kind**: global class  
 
 * [Qualification](#Qualification)
-    * [new Qualification(label, attributes)](#new_Qualification_new)
-    * [.checkQualification(qualification)](#Qualification+checkQualification)
-    * [.setPurpose(purpose)](#Qualification+setPurpose) ⇒ [<code>Qualification</code>](#Qualification)
-    * [.setSourceCategory(sourceCategory)](#Qualification+setSourceCategory) ⇒ [<code>Qualification</code>](#Qualification)
-    * [.setSourceSubCategory(sourceSubCategory)](#Qualification+setSourceSubCategory) ⇒ [<code>Qualification</code>](#Qualification)
-    * [.setSubjects(subjects)](#Qualification+setSubjects) ⇒ [<code>Qualification</code>](#Qualification)
-    * [.toQualification()](#Qualification+toQualification) ⇒ <code>object</code>
+    * [new exports.Qualification(label, attributes)](#new_Qualification_new)
+    * _instance_
+        * [.checkValueAttributes(attributes)](#Qualification+checkValueAttributes)
+        * [.setPurpose(purpose)](#Qualification+setPurpose) ⇒ [<code>Qualification</code>](#Qualification)
+        * [.setSourceCategory(sourceCategory)](#Qualification+setSourceCategory) ⇒ [<code>Qualification</code>](#Qualification)
+        * [.setSourceSubCategory(sourceSubCategory)](#Qualification+setSourceSubCategory) ⇒ [<code>Qualification</code>](#Qualification)
+        * [.setSubjects(subjects)](#Qualification+setSubjects) ⇒ [<code>Qualification</code>](#Qualification)
+        * [.toQualification()](#Qualification+toQualification) ⇒ <code>object</code>
+    * _static_
+        * [.getQualificationByLabel(label)](#Qualification.getQualificationByLabel) ⇒ [<code>Qualification</code>](#Qualification)
 
 <a name="new_Qualification_new"></a>
 
-### new Qualification(label, attributes)
+### new exports.Qualification(label, attributes)
 
 | Param | Type | Description |
 | --- | --- | --- |
 | label | <code>string</code> | The qualification label |
 | attributes | [<code>Qualification</code>](#Qualification) | Qualification's attributes |
 
-<a name="Qualification+checkQualification"></a>
+<a name="Qualification+checkValueAttributes"></a>
 
-### qualification.checkQualification(qualification)
-Check the given qualification respects the following rules:
+### qualification.checkValueAttributes(attributes)
```

Comment:
I do not see how "Value" helps here ? Could this be reduced to checkAttributes ?

Reply:
Well... Yes :+1: